modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java

```diff
@@ -162,6 +172,20 @@ public void onWalCpRecordFsyncStart() {
         walCpRecordFsyncStart = System.currentTimeMillis();
     }
 
+    /**
+     *
+     */
+    public void onSplitAndSortCpPagesStart() {
+        splitAndSortCpPagesStart = System.currentTimeMillis();
```

Comment:
i suggest U.currentTimeMillis() use instead.

Reply:
U.currentTimeMillis gives 10 milliseconds error margin. Also all other methods use System.currentTimeMillis. So we have to use one ore another currentTimeMillsi() for all time logging methods of CheckpointMetricTracker. Do we really have good reason for change?

Reply:
it`s up to u of course, but think it would be more clear.